app/controllers/items_controller.rb

```diff
@@ -28,12 +29,12 @@ def update
     elsif @item.update(item_params)
       redirect_to merchant_item_path(@merchant, @item)
       flash[:notice] = "Item successfully updated."
```

Comment:
If you want to you can make this a one line by doing this; ` redirect_to merchant_item_path(@merchant, @item), notice: "Item successfully updated."`